src/fin/test/gcd_fill.py

```diff
@@ -0,0 +1,13 @@
+from openroad import Design, Tech
+import odb
+
+tech = Tech()
+design = Design(tech)
+
+design.evalTclString("read_db gcd_fill.odb; read_sdc gcd_fill.sdc")
```

Comment:
Why read_db in tcl eval and not through the design object?

Why do we need timing for metal fill?

Reply:
So readDb is not yet on the design object, which I guess, it should be. Will do